### ISSUE TITLE
security key issue

### ISSUE THREAD

**Issue by amar0419** [ant-media/Ant-Media-Server#2979]
While clicking on live broadcast it shows Failed to start. Please check server url and security credentials..but rtmp url is fine..and there is no option for stream key as rtmp url requires stream key for live broadcasting

**Comment by mekya** [comment]
Hi @amar0419 ,

Are you mentioning about LiveVideoBroadcaster? If yes, you need to type the stream name in the editbox

<img src="https://camo.githubusercontent.com/753d14ecec5a22ef9b4d60e1e06f0a0f532306f4f9cf3b3b2ad2e6ac0b3cf9b5/687474703a2f2f616e746d656469612e696f2f77702d636f6e74656e742f75706c6f6164732f323031372f30342f7265636f72642d65313439323335323638373838332e706e67" />

**Comment by amar0419** [comment]
yes, I am talking about LiveVideoBroadcaster...my url is like rtmp_url/stream_key/stream_name and its showing like broadcasting but when I open output url(amazon cloudfront url) on chrome nothing is visible there

**Comment by mekya** [comment]
Hi @amar0419 

Do you use Ant Media Server on your end? If yes, full url including stream_name should be like this
rtmp://SERVER/APP_NAME/STREAM_NAME

Are you sure that you bind the cloudfront url correctly to your origin server?

**Comment by amar0419** [comment]
No, I am not using ant media server..but what I am using its link is link rtmp://server/streamkey/streamtitle

**Comment by pratikkumaar** [comment]
Any fix? of it
I am facing same.
